### PR TITLE
Add poppler package

### PR DESCRIPTION
**Pull request by uberhacker**
Poppler is a PDF rendering library based on the xpdf-3.0 code base.  See https://poppler.freedesktop.org/.

### CODE REVIEW

**Comment by cstrouse** [comment]
When building on x86_64 I get the following error.

```
checking for jpeglib.h... no
configure: WARNING:
There is an installation error in jpeg support. You seem to have only one
of either the headers _or_ the libraries installed. You may need to either
provide correct --with-extra-... options, or the development package of
libjpeg6b. You can get a source package of libjpeg from http://www.ijg.org/
Disabling JPEG support.

checking libjpeg.h works correctly... no
checking libjpeg.h problem can be fixed by XMD_H macro... no, disable libjpeg
configure: error: Install libjpeg before trying to build poppler. You can also decide to use the internal unmaintained DCT decoder or none at all. See --help.
poppler failed to install: `./configure             --prefix=/usr/local             --libdir=/usr/local/lib64` exited with 1
```

**Comment by cstrouse** [comment]
Missing several dependencies:
- libjpeg
- harfbuzz

**Comment by uberhacker** [comment]
Good catch @cstrouse.  I also noticed this package needs libpng and cairo to enable full functionality.  I'll try and update this when I get a chance. Thanks

File: packages/poppler.rb [hunk 1]
```diff
@@ -13,7 +13,6 @@ class Poppler < Package
   })
 
   depends_on 'cairo'
-  depends_on 'fontconfig'
   depends_on 'harfbuzz'
   depends_on 'libjpeg'
```
Comment (cstrouse) [inline packages/poppler.rb]: It also needs `libopenjpeg`.

**Comment by uberhacker** [comment]
Replaced by #1368.

**Pull request closed without merging**